Adding missing release notes (#2789)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.39.0.Final/core.adoc

[id='stunner-data-objects-741']

= Core process engine

> * It is now possible to start a process instance from a given set of node ids (rather than from the normal start node).

Reviewer: ```suggestion
* You can now start a process instance from a given set of node IDs instead of starting a process instance from the start node. 
```
